app/models/dog_searcher.rb

```diff
@@ -1,121 +0,0 @@
-#    Copyright 2017 Operation Paws for Homes
-#
-#    Licensed under the Apache License, Version 2.0 (the "License");
-#    you may not use this file except in compliance with the License.
-#    You may obtain a copy of the License at
-#
-#        http://www.apache.org/licenses/LICENSE-2.0
-#
-#    Unless required by applicable law or agreed to in writing, software
-#    distributed under the License is distributed on an "AS IS" BASIS,
-#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#    See the License for the specific language governing permissions and
-#    limitations under the License.
-
-class DogSearcher
-  PER_PAGE = 30
-
-  ACTIVE_STATUSES = [
-    'adoptable',
-    'adoption pending',
-    'on hold',
-    'return pending',
-    'coming soon'
-  ].freeze
-
-  PUBLIC_STATUSES = ['adoptable', 'adoption pending', 'coming soon'].freeze
-
-  def initialize(params: {}, manager: false)
-    @params = params
-    @manager = manager
-  end
-
-  def search
```

Comment:
very glad to have this mess cleaned up, nice work!

Reply:
I had to clean it up just to understand it and write tests so I didn't break it!